pybliometrics/utils/get_content.py

```diff
@@ -67,30 +67,43 @@ def get_content(url, api, params=None, **kwds):
     from random import shuffle
     from time import sleep, time
 
+    # Get needed ressources for query
     config = get_config()
+
     keys = get_keys()
+
+    # Get tokens and zip with keys
+    insttokens = get_insttokens()
+    insttokens = list(zip(keys, insttokens))
+
+    # Keep keys that are not insttokens
+    keys = keys[len(insttokens):]
+
     session = get_session()
 
-    # Set header, params and proxy
-    try:
-        header = {'X-ELS-APIKey': keys[0],
-                  'Accept': 'application/json',
-                  'User-Agent': user_agent}
-    except IndexError:
-        raise errors[429]
-
-    if config.has_option('Authentication', 'InstToken'):
-        token = config.get('Authentication', 'InstToken')
-        header['X-ELS-Insttoken'] = token
     params = params or {}
     params.update(**kwds)
     proxies = dict(config._sections.get("Proxy", {}))
+    timeout = config.getint("Requests", "Timeout", fallback=20)
 
-    # Replace credentials if provided
-    if "apikey" in params:
-        header['X-ELS-APIKey'] = params.pop("apikey")
+    # Base header
+    base_header = {'Accept': 'application/json',
+                   'User-Agent': user_agent}
+
+    # Get tokens and create header
+    token_key, insttoken = None, None
```

Comment:
Seems unnecessary because it will always be redefined

Reply:
In case there are no `insttokens` found both variables won't be redefined. We need to set them to `None` to avoid an error in [line 116](https://github.com/nils-herrmann/pybliometrics/blob/705736494b4c658ebe230d4429f8b63ea94d0dfc/pybliometrics/utils/get_content.py#L116).

Reply:
oh, now I see, there is an `elif`!
Anyways, if you could simplify this, maybe as if - elif - else statement, that would be great. This function involves a great deal of variables and conditions, every reduction in complexity is a win.